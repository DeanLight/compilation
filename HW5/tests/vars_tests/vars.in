void main(){
	int x = 2;
	bool f3;
	byte y;
	{
		bool f1 = true;
		if(f1){
			printi(y);
			print("\ta\t");
			bool f2 = false;
			if(not f3){
				print("b\t");
				y = 1 b;
				int z = y;
			}
		}
		byte zz = 2 b;
		zz = 3 b;
		printi(y);
	}
	print("\tc\t");
	printi(x);
}