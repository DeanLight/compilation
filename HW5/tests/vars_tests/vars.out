Loaded: ./exceptions.s
0	a	b	1	c	2
